greenlet.js

```diff
@@ -60,8 +60,8 @@ export default function greenlet(asyncFunction) {
 			worker.postMessage([currentId, args], args.filter(x => (
 				(x instanceof ArrayBuffer) ||
 				(x instanceof MessagePort) ||
-				(x instanceof ImageBitmap)
+        (createImageBitmap && x instanceof ImageBitmap)
```

Comment:
I've noticed that `createImageBitmap` raises issues in Safari without the `window` prefix. So this must be `window.createImageBitmap`.